src/preprocessors/customids.md

```diff
@@ -0,0 +1,5 @@
+# Customids
```

Comment:
Maybe “Custom IDs”?

Reply:
To use the content of headings from Markdown sources in the navigation menu, `foliantcontrib.markdown >= 1.0.2` is needed.

`Customids` is the string that MkDocs generates itself according to filename of Markdown source.

Let’s update `requirements.txt`?